packages/devtools_app/lib/src/shared/analytics/_analytics_controller_stub.dart

```diff
@@ -7,5 +7,8 @@ import 'dart:async';
 import 'analytics_controller.dart';
 
 FutureOr<AnalyticsController> get devToolsAnalyticsController => _controller;
-AnalyticsController _controller =
-    AnalyticsController(enabled: false, firstRun: false);
+AnalyticsController _controller = AnalyticsController(
+  enabled: false,
+  firstRun: false,
+  consentMessage: 'fake message',
```

Comment:
nit: since you implemented `fetchAnalyticsConsentMessage,` in  _analytics_stub.dart, can we return `consentMessage: await ga.fetchAnalyticsConsentMessage()` here?